index.js

```diff
@@ -1,9 +1,14 @@
 const dataTransfer = new DataTransfer()
 
+function isDetached(element) {
+  return element.closest('html')
```

Comment:
@bierik, could you clarify the reason behind the function name?

It returns `truthy` value if an element has `html` as one of its parent - which means the element is attached to the DOM, so the function name should be `isAttached`. Though I may miss something :)

Reply:
You are absolutely right the function should be `isAttached`. I changed it. Could you review again and test it?

Reply:
It seems to work fine on my setup. Thanks!